bug: tests may fail periodically

https://github.com/viordash/AtsPLC/actions/runs/11593560044/job/32277856505

```
src/logic_CommonTimer_tests.cpp:118: error: Failure in TEST(LogicCommonTimerTestsGroup, Render_on_bottom_network)
	CHECK_TRUE(testable.Render(frame_buffer, LogicItemState::lisActive, &start_point)) failed

.
src/logic_CommonTimer_tests.cpp:109: error: Failure in TEST(LogicCommonTimerTestsGroup, Render_on_top_network)
	CHECK_TRUE(testable.Render(frame_buffer, LogicItemState::lisActive, &start_point)) failed
```

--- a/Tests_esp8266/src/logic_CommonTimer_tests.cpp
+++ b/Tests_esp8266/src/logic_CommonTimer_tests.cpp
@@ -50,6 +50,7 @@ namespace {
         TestableCommonTimer(uint64_t delay_time_us) : CommonTimer() {
 
             this->delay_time_us = delay_time_us;
+            str_size = sprintf(this->str_time, "%u", (uint32_t)delay_time_us);
         }
         virtual ~TestableCommonTimer() {
         }

